t/apicast-policy-maintenance-mode.t

```diff
@@ -0,0 +1,91 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+repeat_each(1);
+run_tests();
+
+__DATA__
+
+=== TEST 1: Use maintenance mode using default values
+Testing 3 things:
+1) Check default status code
+2) Check default response message
+3) TODO - validate upstream doesn't get the request
```

Comment:
There are several options. You could print something in `upstream` and then check that it does not appear in the output, or you could put and assertion using `luassert`.